server/src/services/Google.ts

```diff
@@ -8,6 +9,27 @@ interface CalendarData {
   description: string;
 }
 
+interface EventData {
+  start?: Date;
+  end?: Date;
```

Comment:
Random question is it possible to have Event without `Date`? like it will have the wrong type of "Date" (because it's `any` in the client)

```suggestion
  start: Date;
  end: Date;
```

Reply:
It's not possible. However when updating attendees date isn't passed as changed, but it's taken from `oldEventData`. https://github.com/freeCodeCamp/chapter/blob/c9100ae387e66622409b1cfa24e7d1246538270c/server/src/services/Google.ts#L118-L123

Reply:
Thank you for clearing this